docs/source-code-overview.CN.md

```diff
@@ -0,0 +1,221 @@
+# KVRocks源码概览
```

Comment:
```suggestion
# Kvrocks源码概览
```
It really is a big trouble for us how to spell kvrocks, but we currently, we still prefer to use **Kvrocks** when we need upper case.

Reply:
Done